dev/com.ibm.ws.channelfw/src/com/ibm/ws/tcpchannel/internal/ChannelSelector.java

```diff
@@ -18,9 +18,11 @@
 import java.nio.channels.SocketChannel;
```

Comment:
Can we update the copyright range here?

Reply:
That range is accurate.  I actually wrote it that long ago.